src/views/Main/Link/Link.js

```diff
@@ -0,0 +1,316 @@
+import React from 'react';
+import { Button } from 'react-bootstrap';
+import Select from 'react-select';
+import Promise from 'bluebird';
+import { EntityDataModelApi } from 'loom-data';
+import Page from '../../../components/page/Page';
+import buttonStyles from '../../../core/styles/buttons.css';
+import styles from './styles.module.css';
+
+export class Link extends React.Component {
+
+  constructor() {
+    super();
+    this.state = {
```

Comment:
Right, I just figured since this needed to be in by/before Tuesday and since most of it will probably need to be rewritten anyway once we have designs, it was better to just get something up and running. I can try to move it over to redux, but I'm not sure if I'll have enough time before the demo to set that up correctly.